docs/howto/how_to_debug.md

```diff
@@ -71,7 +71,7 @@ Start the module to be debugged, either by using the command line or by using
 the Dreamview interface. The following is an example of debugging the
 **Planning** module from the Dreamview interface.
 
-- Open Chrome browser, enter the URL: http://localhost:8888/
+- Open Chrome browser, enter the URL: <http://localhost:8888/>
 - Open the Dreamview interface
```

Comment:
We don't say enter the url, we say `Open http://xxxx in your favorite web browser (e.g. Chrome).`